src/Common/tests/System/IO/Compression/ZipTestHelper.cs

```diff
@@ -115,7 +115,14 @@ public static void StreamsEqual(Stream ast, Stream bst, int blocksToRead)
                 ac = ast.Read(ad, 0, 4096);
                 bc = bst.Read(bd, 0, 4096);
 
-                Assert.Equal(ac, bc);
+                if (ac != bc)
+                {
+                    string rep = Text.Encoding.Default.GetString(bd);
+                    rep = rep.Replace("\r\n", "\n");
+                    rep = rep.Replace("\n", "\r\n");
```

Comment:
might make sense to extract this to NormalizeLineEndings

Reply:
Good call, done.